src/Sarif.Multitool/Sarif.Multitool.csproj

```diff
@@ -2,11 +2,29 @@
   
   <PropertyGroup>
     <OutputType>Exe</OutputType>
+
+    <!-- To Publish single-file exes, run: 
+      dotnet publish Sarif.Multitool.csproj -c Release -f netcoreapp3.0 -r win-x64
+      dotnet publish Sarif.Multitool.csproj -c Release -f netcoreapp3.0 -r linux-x64
+      dotnet publish Sarif.Multitool.csproj -c Release -f netcoreapp3.0 -r osx-x64
+
+      For settings which we want only for these builds, we use [Condition="'$(RuntimeIdentifier)' != ''"] to identify them
```

Comment:
> F [](start = 5, length = 2)

Suggestion: "We use [Condition="'$(RuntimeIdentifier)' != ''"] to identify settings that are used only for these builds."

Reply:
Good rewording - fixed.